app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,14 @@
+    <div class="navbar">
+        <div class="logo">
+          <%= vite_image_tag "images/logos/logo_text_white.png", width: "180px" %>
+        </div>
+
+        <div class="links">
+          <ul>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Sessions</a></li>
+            <li><a href="#">Contact</a></li>
+            <li><a href="#">Videos</a></li>
+          </ul>
+        </div>
+    </div>
```

Comment:
Again the indentation should be 2 spaces, not 4
Also because your ul is a list of links you should use a <nav> tag instead. And inside the nav you can just use <a> tags directly wihout <li>